schema/synthetix/view_trades.sql

```diff
@@ -0,0 +1,55 @@
+CREATE OR REPLACE VIEW synthetix.view_trades (
+    block_time,
+    project,
+    version,
+    trader_a,
+    trader_b,
+    token_a_amount_raw,
+    token_b_amount_raw,
+    token_a_address,
+    token_b_address,
+    exchange_contract_address,
+    tx_hash,
+    trace_address,
+    evt_index,
+    token_a_amount,
+    token_b_amount,
+    token_a_usd_price,
+    token_b_usd_price
+) AS SELECT
+    trade.evt_block_time AS block_time,
+    'Synthetix' AS project,
+    '1' AS version,
+    trade.account AS trader_a,
+    NULL::bytea AS trader_b,
+    trade."fromAmount"/1e18 AS token_a_amount_raw,
+    trade."toAmount"/1e18 AS token_b_amount_raw,
+    synths_a.address AS token_a_address,
+    synths_b.address AS token_b_address,
+    trade.contract_address AS exchange_contract_address,
+    trade.evt_tx_hash AS tx_hash,
+    NULL::integer[] AS trace_address,
+    trade.evt_index AS evt_index,
+    trade."fromAmount"/1e18 * rates_a.currency_rate/1e18 as token_a_amount,
+    trade."toAmount"/1e18 * rates_a.currency_rate/1e18 as token_b_amount,
+    rates_a.currency_rate/1e18 as token_a_usd_price,
+    rates_b.currency_rate/1e18 as token_b_usd_price
+FROM
+    synthetix."Synthetix_evt_SynthExchange" trade
+LEFT JOIN synthetix."view_synths" synths_a
```

Comment:
I am not sure LEFT JOINS are the most performant way to combine the data but it should be better than the lateral joins, right? Perhaps just a subquery for the data... Let me know what you think